project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.4.9
+sbt.version=1.3.13
```

Comment:
why is sbt downgraded?

Reply:
Actually one of us were having issues running the repo on local, so had to downgrade sbt and unfortunately it got pushed along. I have undone this change.

Reply:
Please also double check the BSP file has been deleted if one is generated once you upgrade